### PR TITLE
IN_AT_02.04.017| Homepage(Dashboard)/Check side panel sub-menu with 5 items

### PR DESCRIPTION
**Pull request by InnaNMakarova**
https://trello.com/c/ZGDPToZs/1417-at0204017-homepagedashboard-check-side-panel-sub-menu-with-5-items

### CODE REVIEW

**Comment by DariaMas7** [comment]
Hello. we name our branch with IniTials of our Last/First name -> and then brief description of the task

File: cypress/e2e/sidePanelSubMenu_zb.cy.js [hunk 1]
```diff
@@ -72,5 +72,13 @@ describe('Side panel sub menu', () => {
         cy.get('a[href="/manage"] .task-link-text').should('have.text', dashboardDropdownItems[3])
         cy.get('a[href="/me/my-views"] .task-link-text').should('have.text', dashboardDropdownItems[4])              
     })
+    
+    it.only('AT_02.04.017| Homepage(Dashboard)/Check side panel sub-menu with 5 items', function (){
```
Comment (DariaMas7) [inline cypress/e2e/sidePanelSubMenu_zb.cy.js:76]: remove only please

**Comment by NatalyaVasylyeva** [comment]
Hello. You forgot to write the test case in your TC__02.04.017| Homepage(Dashboard)/Check side panel sub-menu with 5 items. Please, write preconditions, steps, expected results

File: cypress/e2e/sidePanelSubMenu_zb.cy.js (hunk 1 above)
Comment (InnaNMakarova) [reply cypress/e2e/sidePanelSubMenu_zb.cy.js:76]: Ох, я накосячила(, спасибо, сейчас все доделаю-переделаю

**Review by LenaAndreeva — APPROVED** [review]

**Pull request merged**